modules/402-ingress-nginx/crds/ingress-nginx.yaml

```diff
@@ -906,6 +906,13 @@ spec:
                   description: |
                     Switch off HTTP2 support.
                   default: false
+                disableHTTP3:
+                  type: boolean
+                  description: |
+                    Switch off HTTP3 support.
+                    
+                    HTTP3 is only available in ingress-nginx version 1.10.
+                  default: false
```

Comment:
@ipaqsa what if we add an additional x-kubernetes-validation rule for the disableHTTP3 flag [here](https://github.com/deckhouse/deckhouse/blob/339dea1643a1a4f59a3268a447eab5b4b6d76683/modules/402-ingress-nginx/crds/ingress-nginx.yaml#L1104-L1106) to make sure users don't get messed up with HTTP3 settings for controllers others than 1.10?

Reply:
I patched the controller configuration and nginx template to add useHTTP3 parameter, it is not present in previous versions.

Reply:
Anyway, it can lead to some misunderstanding, so we have to add `oneOf` or `x-kubernetes-validation` to avoid invalid configuration with wrong expectations

Reply:
Also I think we need to enable it explicitly, not disable